presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestQuantileDigestAggregationFunction.java

```diff
@@ -44,55 +39,21 @@
 import static com.facebook.presto.operator.aggregation.FloatingPointBitsConverterUtil.floatToSortableInt;
 import static com.facebook.presto.operator.aggregation.TestMergeQuantileDigestFunction.QDIGEST_EQUALITY;
 import static com.facebook.presto.spi.type.BigintType.BIGINT;
-import static com.facebook.presto.spi.type.BooleanType.BOOLEAN;
 import static com.facebook.presto.spi.type.DoubleType.DOUBLE;
 import static com.facebook.presto.spi.type.RealType.REAL;
-import static com.facebook.presto.spi.type.TypeSignature.parseTypeSignature;
+import static com.facebook.presto.spi.type.StandardTypes.QDIGEST;
 import static com.facebook.presto.sql.analyzer.TypeSignatureProvider.fromTypes;
 import static com.google.common.collect.ImmutableList.toImmutableList;
 import static java.lang.Double.NaN;
-import static java.lang.Integer.max;
-import static java.lang.Integer.min;
-import static java.lang.String.format;
 
 public class TestQuantileDigestAggregationFunction
-        extends AbstractTestFunctions
+        extends TestStatisticalDigestAggregationFunction
 {
-    private static final Joiner ARRAY_JOINER = Joiner.on(",");
-    private static final MetadataManager METADATA = MetadataManager.createTestMetadataManager();
+    private static final double STANDARD_ERROR = 0.01;
 
-    @Test
-    public void testDoublesWithWeights()
```

Comment:
Why do you remove tests in `TestQuantileDigestAggregationFunction`?

Reply:
I have the test in `TestStatististicalDigestAggregationFunction`, since it was a repeated test for both `QuantileDigest` and `TDigest`.